### PR TITLE
Refresh

### PR DESCRIPTION
**Pull request by talimarcus**
Design refresh (spec here: https://ge.ent.box.com/notes/185167242781?s=sk8uem5vb8x4sd2c3sywxldbfx3qfktl) 

Note: Wade confirmed that it's okay to use to use a popover instead of a dropdown (as illustrated in the spec).

### CODE REVIEW

File: sass/px-login.scss [hunk 1]
```diff
@@ -39,33 +39,37 @@ $inuit-enable-btn--bare : true;
...
 
-.px-login-user-button {
-    width: 180px;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
+.login-button--chevron-icon {
+  width: 12px;
+  min-width: 12px;
+  margin-left: 5px;
```
Comment (katemenkhaus) [inline sass/px-login.scss]: with margin's we generally use $inuit-base-spacing-unit.

File: sass/px-login.scss [hunk 2]
```diff
@@ -39,33 +39,37 @@ $inuit-enable-btn--bare : true;
...
+  margin-left: auto;
+  margin-right: auto;
+  align-items: center;
 }
 
-.px-login-button {
-    color: var(--px-login-color, $gray13);
+.login-button--user-icon {
+  min-width: 30px;
+  height: 30px;
```
Comment (katemenkhaus) [inline sass/px-login.scss]: I would also use $inuit-base-spacing-unit here because you want it to be dependent on the font size

**Comment by katemenkhaus** [comment]
Did you double check that you needed all the imports in the sass?

**Comment by katemenkhaus** [comment]
also Martin mentioned that sense our components aren't getting qa we need to make sure we have enough unit tests.

**Comment by talimarcus** [comment]
@Menkhaus-ge:

- updated to use $inuit-base-spacing-unit and calculateRem()
- removed unnecessary sass imports
- checked the test coverage, looks good to me

**Comment by talimarcus** [comment]
@Menkhaus-ge one last review please? :)

**Comment by katemenkhaus** [comment]
Looks good

**Pull request merged**